imports/api/crud-controller/module/methods.js

```diff
@@ -46,6 +59,12 @@ export const insertOneModuleInSemester = function insertOneModuleInSemester(seme
   if (!retrievedSemesters) {
     return {};
   }
+
+  // checks semester index is not larger than the size of semester
```

Comment:
What does this check for again? Or perhaps this is defensive programming in case the index of a sem is larger than its actual size?